tldr/cross_file_calls.py

```
@@ -3939,3 +3965,1019 @@ def _build_php_call_graph(
+                # Don't descend into nested definitions — visit_all walks them
+                # separately, and attributing their calls to the outer function
+                # would double-count.
+                if child.type in def_node_types:
+                    continue
+                visit(child)
+
+        visit(body_node)
+        return calls
+
+    # Pass 1: collect all top-level definitions so forward references resolve to "intra".
+    def collect_defs(node):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `collect_defs`

Add return type annotation: `None`

(ANN202)